config/locales/instructions/en.yml

```diff
@@ -56,6 +56,11 @@ en:
         intro: 'Password strength: '
         iv: Good
         v: Great!
-    personal_key_accent: Write it down or print it out.
-    personal_key_html: This is the only way to regain access to your account if you
-      lose the phone where we send your security code. %{accent}
+    personal_key:
+      accent: Write it down or print it out.
+      info_html: This is the only way to regain access to your account if you lose
+        your password or phone. %{accent}
```

Comment:
I don't think this is accurate. If you lose your password, the personal key will not let you in in lieu of your password. And if you only lose your password, all you need to do to get back in is reset it. The only time the personal key comes into play after resetting your password is if you are a verified user. You need your personal key in order to maintain your verified status. If you don't have your personal key, then you will have to verify your identity again.

Reply:
Good point, in the current screen the text is: "This is the only way to regain access to your account if you lose the phone where we send your security code." I dunno if that's conditional on your MFA method choice or if it's unfortunately hard-coded, but it's more accurate than the current text.

Reply:
(just checked, we do not conditionally update it)

Steve, please change the info_html text here back to the text I copied above from the current screen, I think that is more correct.

Reply:
Yeah, we didn't update the text after we started offering more 2FA choices.

Reply:
I noticed the wording was off wrt the password but thought it was not only benign but  increased the urgency of writing it down.  However I never liked the 'where we send your security code'.  This seems to add confusion wrt the other phone based codes that are sent and I thought that was the rationale behind part of the wording change.  Can we at least drop that phrase?

Reply:
To reiterate 'where we send your security code' is confusing.  It almost implies we are sending this personal key.

Reply:
All good points. How about: "This is the only way to regain access to your account if you lose your phone or security token."

Phone covers calling, texting, and auth app, and security token is vague enough for FIDO or piv/cac.

Reply:
I don't love the term security token. I feel like people using piv/cac will not put the two together. It seems to me that we just need a better generic term for 2FA options. We consistently call 2FA options "security options". Might be worth rethinking at some point if we want to change that naming convention.

Reply:
Will this work? "This is the only way to regain access to your account if you lose your phone or security options."

Reply:
works for me for now.